HTAN.model.csv

```diff
@@ -7,6 +7,11 @@ Patient,HTAN patient,,"Component, HTAN Participant ID",,FALSE,Individual Organis
 File,A type of Information Content Entity specific to OS,,,,FALSE,Information Content Entity,,https://w3id.org/biolink/vocab/DataFile,
 Filename,Name of a file,,,,TRUE,,,,regex search ^.+\/\S*$
 File Format,"Format of a file (e.g. txt, csv, fastq, bam, etc.)","hdf5, bedgraph, idx, idat, bam, bai, excel, powerpoint, tif, tiff, OME-TIFF, png, doc, pdf, fasta, fastq, sam, vcf, bcf, maf, bed, chp, cel, sif, tsv, csv, txt, plink, bigwig, wiggle, gct, bgzip, zip, seg, html, mov, hyperlink, svs, md, flagstat, gtf, raw, msf, rmd, bed narrowPeak, bed broadPeak, bed gappedPeak, avi, pzfx, fig, xml, tar, R script, abf, bpm, dat, jpg, locs, Sentrix descriptor file, Python script, sav, gzip, sdf, RData, hic, ab1, 7z, gff3, json, sqlite, svg, sra, recal, tranches, mtx, tagAlign, dup, DICOM, czi, mex, cloupe, am, cell am, mpg, m, mzML,scn, dcc, rcc, pkc, sf, bedpe",,,TRUE,,,,
+CDS Genomic Reference ,One or more characters used to identify the published NCBI genetic sequence that is used as a reference against which other sequences are compared.,"hg19, hg38_long_polya, Release 31 (GRCh38.p12)",,,TRUE,Sequencing,,https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=alignment_workflow,

```

Comment:
This does not appear to be the right source

Reply:
It looks like the CDE specified in the mapping (11529031) does not actually have any PVs but rather in our mapping we were asked to perform some minimal harmonization.|

I would remove the valid values here and add a `str` validation rule